src/sentry/web/client_config.py

```diff
@@ -122,6 +122,13 @@ def get_client_config(request=None):
     Provides initial bootstrap data needed to boot the frontend application.
     """
     if request is not None:
+        customer_domain = None
+        if hasattr(request, "subdomain") and request.subdomain:
```

Comment:
nit: Is the second check for `and request.subdomain` redundant since you check `hasattr` just before that?

Reply:
Ohhh wait, I get it nvm lol. if `subdomain  = None` hasattr still returns true, my bad!